File: Stage15/C/Problem13241.c
#include <stdio.h>

long long gcd(long long a, long long b) {
    while (b != 0) {
        long long temp = a % b;
        a = b;
        b = temp;
    }
    return a;
}

long long lcm(long long a, long long b) {
    return a * b / gcd(a, b);
}

int main(void) {
    long long A, B;
    scanf("%lld %lld", &A, &B);

    printf("%lld\n", lcm(A, B));
    return 0;
}

File: Stage15/Java/Problem13241.java
package Stage15.Java;

import java.math.BigInteger;
import java.util.Scanner;

public class Problem13241 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        long a = sc.nextLong();
        long b = sc.nextLong();

        BigInteger A = BigInteger.valueOf(a);
        BigInteger B = BigInteger.valueOf(b);

        BigInteger gcd = A.gcd(B);
        BigInteger lcm = A.multiply(B).divide(gcd);

        System.out.println(lcm);
        sc.close();
    }
}
